### ISSUE TITLE
warning: lou_getDataPath is deprecated

### ISSUE THREAD

**Issue by sthibaul** [liblouis/liblouisutdml#97]
Hello,

The testsuite prints this warning

```
warning: lou_getDataPath is deprecated.
```

Which is apparently new in the newer release of liblouis.

Samuel

**Comment by sthibaul** [comment]
(as well as mere `file2brl` invocations)

**Comment by bertfrees** [comment]
Yes this should be updated. https://github.com/liblouis/liblouis/pull/1317 explains how.

**Comment by egli** [comment]
Bert says in a mail that

> perhaps the easiest is to register a custom table resolver function

I don't have time to do this for this release. PRs welcome.
